storybook/stories/FileView.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { CSSProperties } from "react"
+import { storiesOf } from "@storybook/react"
+import { FilesMokedData } from '../moked/files'
+import { FileView, ImageView } from "../../packages/ui"
+
+
+storiesOf("FileView", module)
+  .add("ImageView", () => (
+    <div className="w-24 h-24">
+      <FileView file={FilesMokedData[0]}/>
+    </div>
+  ))
+  .add("AudioView", () => (
+    <div className="w-24 h24">
```

Comment:
```suggestion
    <div className="w-24 h-24">
```
The player is not visible anyway. And the files in the mock are not audio or video, so you can't see the story correctly.